### ISSUE TITLE
Higher learning rate - No sound after model inference

### ISSUE THREAD

**Issue by KatieBelli** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#798]
I changed the learning rate from 1e-4 to 5e-3 and trained for 350 epochs but no matter which audio file I infer with the model there is always no sound. How can I fix this?

**Comment by aizawa-naoki** [comment]
I think it is because 5e-3 = 1e-4 * **50** , so high for break the model
